lib/bootloader_setup.pm

```diff
@@ -64,120 +78,154 @@ sub select_bootmenu_option {
+        elsif (check_var('ARCH', 'ppc64le')) {
+            send_key_until_needlematch 'inst-oninstallation', 'up', 10, 5;
+        }
+        elsif (!get_var('JEOS')) {
+            send_key_until_needlematch 'inst-oninstallation', 'down', 10, 5;
         }
     }
     return 0;
 }
 
 sub bootmenu_default_params {
-    # https://wiki.archlinux.org/index.php/Kernel_Mode_Setting#Forcing_modes_and_EDID
-    if (check_var('VIRSH_VMM_FAMILY', 'hyperv')) {
-        type_string_slow " video=hyperv_fb:1024x768";
+    if (check_var('ARCH', 'ppc64le')) {
+        # edit menu, wait until we get to grub edit
+        wait_screen_change { send_key "e" };
+        # go down to kernel entry
+        send_key "down";
+        send_key "down";
+        send_key "down";
+        wait_screen_change { send_key "end" };
+        # load kernel manually with append
+        if (check_var('VIDEOMODE', 'text')) {
+            type_string_very_slow " textmode=1";
+        }
+        type_string_very_slow " Y2DEBUG=1";
     }
-    type_string_slow "Y2DEBUG=1 ";
+    else {
+        # https://wiki.archlinux.org/index.php/Kernel_Mode_Setting#Forcing_modes_and_EDID
+        if (check_var('VIRSH_VMM_FAMILY', 'hyperv')) {
+            type_string_slow " video=hyperv_fb:1024x768";
+        }
+        type_string_slow "Y2DEBUG=1 ";
 
-    # gfxpayload variable replaced vga option in grub2
-    if (!is_jeos && !is_casp && (check_var('ARCH', 'i586') || check_var('ARCH', 'x86_64'))) {
-        type_string_slow "vga=791 ";
-        type_string_slow "video=1024x768-16 ";
-        assert_screen check_var('UEFI', 1) ? 'inst-video-typed-grub2' : 'inst-video-typed', 4;
-    }
+        # gfxpayload variable replaced vga option in grub2
+        if (!is_jeos && !is_casp && (check_var('ARCH', 'i586') || check_var('ARCH', 'x86_64'))) {
+            type_string_slow "vga=791 ";
+            type_string_slow "video=1024x768-16 ";
+            assert_screen check_var('UEFI', 1) ? 'inst-video-typed-grub2' : 'inst-video-typed', 4;
+        }
 
-    if (!get_var("NICEVIDEO") && !is_jeos) {
-        type_string_very_slow "plymouth.ignore-serial-consoles ";    # make plymouth go graphical
-        type_string_very_slow "linuxrc.log=$serialdev ";             # to get linuxrc logs in serial
-        type_string_very_slow "console=$serialdev ";                 # to get crash dumps as text
-        type_string_very_slow "console=tty ";                        # to get crash dumps as text
+        if (!get_var("NICEVIDEO") && !is_jeos) {
+            type_string_very_slow "plymouth.ignore-serial-consoles ";    # make plymouth go graphical
+            type_string_very_slow "linuxrc.log=$serialdev ";             # to get linuxrc logs in serial
+            type_string_very_slow "console=$serialdev ";                 # to get crash dumps as text
+            type_string_very_slow "console=tty ";                        # to get crash dumps as text
```

Comment:
I meant you can rebase locally, i.e. fetch changes from @mnowaksuse and we can merge all commits in this PR then at once